### ISSUE TITLE
summingbird-example is broken

### ISSUE THREAD

**Issue by johnynek** [twitter/summingbird#643]
Looks like we are transitively pulling a later (3.4.x) branch of zookeeper, but storm 0.9.0-wip15 needs 3.3, so it fails at runtime.

It may work to exclude zookeeper from where we are getting it (maybe finagle via storehaus memcache or something).

**Comment by ianoc** [comment]
Its odd we ran into this and our tests passed, it starts up a local ZK to run storm in local mode for the tests....

**Comment by johnynek** [comment]
I assume that something in the example is exercising in a way that we aren't in the storm tests (for instance, we use storehaus-memcache in the example).

**Comment by ianoc** [comment]
Yeah, though we have a storehaus dependency everywhere, and it all depends on util. (Though maybe not finagle i suppose). Just a little concerned about the utility of the tests. But i guess transitive dependency hell is what it is...
